safe/gui/tools/minimum_needs/needs_profile.py

```diff
@@ -288,15 +288,19 @@ def root_directory(self):
         :rtype: QString
         """
         if self._root_directory is None or self._root_directory == '':
+            # Map the root directory to ~/.qgis2/inasafe so the minimum needs
+            # profile will be placed there (~/.qgis2/inasafe/minimum_needs).
             try:
                 # noinspection PyArgumentList
-                self._root_directory = QgsApplication.qgisSettingsDirPath()
+                self._root_directory = os.path.join(
+                    QgsApplication.qgisSettingsDirPath(),
+                    'inasafe')
             except NameError:
                 # This only happens when running only one test on its own
                 self._root_directory = None
             if self._root_directory is None or self._root_directory == '':
                 self._root_directory = os.path.join(
-                    os.path.expanduser('~'), '.qgis2')
+                    os.path.expanduser('~'), '.qgis2', 'inasafe')
```

Comment:
Yes, it will be a QGIS bug.
I don't mind to put the code there. Perhaps, someone from the histroy got the bug then he put the code.

+0 from me for remove or not to remove it :)